docs/DeveloperGuide.md

```diff
@@ -275,31 +275,35 @@ The target users...
+| `* * *` | user                     | access a patient's contact number                                      | quickly get in contact with the patient                               |
+| `* * *` | user                     | access a patient's age                                                 | provide age-specific care and treatment to the patient                |
+| `* *`   | user                     | filter patient records by criteria such as medical condition and age   | simplify the process of finding specific patient information          |
+| `*`     | user                     | customise tags for my patients                                         | have control over the organisation of my patients                     |
+| `*`     | user                     | group delete patients by category or tag                               | save time when removing large amount of patient records               |
+| `* *`   | user                     | search for specialists by their speciality                             | refer patients to the right experts                                   |
+| `*`     | user with poor eyesight  | customise font size and style of the App                               | easily read the information on the screen                             |
+| `*`     | user who is colour blind | customise colour schemes of the App                                    | meet my accessibility needs                                           |
+| `*`     | user who is a fast typer | perform all tasks will the CLI rather than the GUI                     | be more efficient in managing records                                 |
+| `* *`   | user who is impatient    | add new keyboard shortcuts to the App                                  | save time when performing frequently repeated tasks                   |
+| `*`     | lazy user                | import App data previously stored as a .CSV file directly into the App | save time on setting up the application                               |
+| `*`     | lazy user                | export App data                                                        | share the data easily when required                                   |
+| `* * *` | user                     | save the App data                                                      | retain data for future reference                                      |
+| `* * *` | user                     | load previously saved App data                                         | examine past records/information                                      |
+| `*`     | experienced user         | manipulate the save file directly                                      | make quick changes to data without having to open the application     |
+| `* * *` | user                     | add a new specialist to the records                                    | keep track of details of new specialists patients can be referred  to |
```

Comment:
There seems to be an erroneous extra whitespace after `referred` in the  benefits column.